package.json

```diff
@@ -17,8 +17,9 @@
     "lint": "eslint . --format eslint-formatter-multiple",
     "format": "prettier --write .",
     "lint:fix": "npm run lint -- --fix",
-    "test": "jest",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test:unit": "jest --config ./jest.unit.config.js ./src",
+    "test:e2e": "jest --config ./jest.e2e.config.js ./e2e/tests"
```

Comment:
Need to add "test" and call test:unit and test:e2e there for the CI to run both for PR's.

```
 echo 'Error: no test specified'
```